apps/ios/Shared/Views/Chat/ChatItem/CIFileView.swift

```diff
@@ -125,10 +125,11 @@ struct CIFileView: View {
                 switch file.fileProtocol {
                 case .xftp: progressView()
                 case .smp: fileIcon("doc.fill")
+                case .local: fileIcon("doc.fill")
                 }
             case let .sndTransfer(sndProgress, sndTotal):
                 switch file.fileProtocol {
-                case .xftp: progressCircle(sndProgress, sndTotal)
+                case .xftp, .local: progressCircle(sndProgress, sndTotal)
```

Comment:
same as on android - can't be in progress (transferred), can't be waiting (accepted)